example/ExampleMaterialDesignControls/ViewModels/MaterialPickerViewModel.cs

```diff
@@ -64,12 +65,46 @@ public MaterialPickerViewModel()
                 }
             };
 
-            ItemsSource = new ObservableCollection<string> { "Model 1", "Model 2", "Model 3", "Model 4" };
-            SecondaryItemsSource = new List<string> { "A", "B", "C", "D" };
-            SelectedItem = "Model 2";
-            SecondarySelectedItem = "C";
+            ItemsSource = new ObservableCollection<Country>

```

Comment:
I like it the sample, but from my point of view, the second item source should change when you change the first item, so I’m not sure if we can do it with countries and cities.
Searching I have found two samples that we could use in our samples:
* https://i.stack.imgur.com/iYrTW.jpg
* https://augmentedcodesite.files.wordpress.com/2022/02/2022-02-07-wheel.png
Also the samples with double picker are running a bit strange on iOS, on both columns shows countries and when I have added new data appear a lot of new items in the right column.